kubernetes-csi-driver-nfs.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: kubernetes-csi-driver-nfs
+  version: 4.10.0
+  epoch: 0
+  description: This driver allows Kubernetes to access NFS server on Linux node
+  copyright:
+    - license: Apache-2.0
```

Comment:
I think we'll require some runtime deps here (reading: https://github.com/kubernetes-csi/csi-driver-nfs/blob/b68ee7181160b12717c80c53667e2d33bf1edc7b/Dockerfile#L21 and checking `crane export gcr.io/k8s-staging-sig-storage/nfsplugin:canary nfsplugin.tar` and `tar -tf nfsplugin.tar`) 💭